packages/zefyr/lib/src/widgets/controller.dart

```diff
@@ -205,4 +205,36 @@ class ZefyrController extends ChangeNotifier {
       composing: TextRange.empty,
     );
   }
+
+  // ノート最後尾に改行を追加
+  void addNewlineAtLast() {
+    replaceText(
+      document.length - 1,
+      0,
+      '\n',
+      selection: selection.copyWith(
+        baseOffset: selection.baseOffset + 1,
+        extentOffset: selection.baseOffset + 1,
+      ),
+    );
```

Comment:
これだと、改行だけされて、selectionの位置が変わらなそうです

なので、このあとに updateSelectionAtLast()を実行してもいいかもです！
```suggestion
    replaceText(
      document.length - 1,
      0,
      '\n',
    );
```

Reply:
あれ、ここの部分でselectionも更新して改行の下に入ってる認識だったんですが、1回目のタップで改行だけされて、selectionの位置が変わらないってことですかね？
となると、こいつって何者なんだ..
```
      selection: selection.copyWith(
        baseOffset: selection.baseOffset + 1,
        extentOffset: selection.baseOffset + 1,
      ),
```
Uploading Screen Recording 2021-06-16 at 18.44.56.mov…

Reply:
ここで更新はできるのですが、この書き方だと、

- selectionをいまの位置から1文字だけ右にずらす

みたいな操作になっていそうですね

もしそうするのであれば、
 updateSelectionAtLast() で更新している位置と同じ位置にselectionをcopyWithし直す必要があります

Reply:
@MasayukiYAMA 修正しました！